### ISSUE TITLE
settings.gradle pluginManagement.repositories should support mavenCentral() shortcut

### ISSUE THREAD

**Issue by JLLeitschuh** [gradle/gradle#3169]
### Expected Behavior

In my `settings.gradle` I should be able to do this:
```gradle
pluginManagement {
    repositories {
        mavenCentral()
    }
}
```
Similar to how I can do this in other parts of my build:
```gradle
repositories {
    mavenCentral()
}
```

### Current Behavior
This the above example throws the following exception:
```
* What went wrong:
A problem occurred evaluating settings 'samples'.
> Could not find method mavenCentral() for arguments [] on object of type org.gradle.plugin.repository.internal.DefaultPluginRepositoriesSpec.
```

### Context
I would like to be able to use `pluginManagement` easily with plugins authors have published to `mavenCentral` or `jcenter` easily.

### Your Environment
```
------------------------------------------------------------
Gradle 4.2
------------------------------------------------------------

Build time:   2017-09-20 14:48:23 UTC
Revision:     5ba503cc17748671c83ce35d7da1cffd6e24dfbd

Groovy:       2.4.11
Ant:          Apache Ant(TM) version 1.9.6 compiled on June 29 2015
JVM:          1.8.0_92 (Oracle Corporation 25.92-b14)
OS:           Mac OS X 10.12.6 x86_64

```

**Comment by oehme** [comment]
@eskatos This relates to what we discussed the other day - changing `pluginManagement` to using a normal `RepositoryHandler` instead, as soon as the portal can be used as a normal Maven repo.

**Comment by eskatos** [comment]
This has been delivered as part of #3347 and will be included in Gradle 4.4-RC1
See https://github.com/gradle/gradle/commit/46675fb8df66325ce42b6f3af6e62bc5d25ef167#diff-22ccc36995eafb5a62f43ac87098214fR272

Closing

**Comment by livanov** [comment]
Doesn't work with 4.7-all
